src/lib/src/viewer/viewer.component.scss

```diff
@@ -44,25 +44,24 @@
   fill-opacity: 0; /* Svg-tile filled black if this is not set? */
 }
 
-
-mime-viewer-header {
+.navbar {
   height: 64px;
   position: absolute;
-  top: 0;
   width: 100%;
-  overflow:hidden;
+  overflow: hidden;
   z-index: 2;
 }
 
-mime-viewer-footer {
-  height: 64px;
+.navbar__header {
+  top: 0;
+  width: 100%;
+}
+
+.navbar__footer {
```

Comment:
Because support is so inconsistent between current browsers as well as older releases, authors are strongly advised to avoid using underscores in class and ID names. A common substitute is the hyphen character